server/test/models/ProgramModelTest.java

```diff
@@ -93,6 +96,8 @@ public void canSaveProgram() throws UnsupportedQuestionTypeException {
     ProgramModel program = new ProgramModel(definition);
 
     program.save();
+    appStatusRepo.createOrUpdateStatusDefinitions(
+        program.getProgramDefinition().adminName(), new StatusDefinitions());
```

Comment:
throughout this file, I think we can just remove all mention of statuses instead of also creating status definitions to assert

Reply:
Right now, without status, the programbuilder class is complaining. Let me tackle it in the next PR.